cpp-package/example/test_ndarray.cpp

```diff
@@ -0,0 +1,58 @@
+/*

```

Comment:
Can we use a more descriptive file name? test_ndarray_copy.cpp?

Reply:
Already fixed.